### PR TITLE
Fabo/pick first network

### PR DESCRIPTION
**Pull request by faboweb**
If using a local network with limited active networks this would sometimes crash the FE

### CODE REVIEW

File: app/src/vuex/modules/connection.js [hunk 1]
```diff
@@ -51,20 +51,11 @@ export default function ({ apollo }) {
...
-          // remove additional execution of checkForPersistedNetwork
-          await dispatch(`setNetwork`, defaultNetwork)
-          await commit(`setNetworkSlug`, defaultNetwork.slug)
-        } else {
-          // otherwise we connect to a fallback network
-          const fallbackNetwork = state.networks.find(
-            (network) => network.id == state.externals.config.fallbackNetwork
-          )
-          await dispatch("setNetwork", fallbackNetwork)
-        }
```
Comment (Bitcoinera) [inline app/src/vuex/modules/connection.js:67]: quite a simplification here

**Comment by Bitcoinera** [comment]
Why cannot we see the checks in here?

**Comment by faboweb** [comment]
> Why cannot we see the checks in here?

weird

**Comment by faboweb** [comment]
🚀 Deployed on https://5f0f49036cae8aebff061bc2--lunieio.netlify.app

**Pull request merged**